modules/serverless-logic-deploying-workflows-dev-mode.adoc

```diff
@@ -102,23 +66,48 @@ $ oc edit sonataflow <workflow_name> -n <your_namespace>
 +
 [source,terminal]
 ----
-$ oc get sonataflows -n <your_namespace>
+$ oc get sonataflows -n <namespace>
+----
+
+.. Find the pod of your application by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -n <namespace>
 ----
 
 .. View the workflow logs by running the following command:
 +
 [source,terminal]
 ----
-$ oc logs <workflow_pod_name> -n <your_namespace>
+$ oc logs <workflow_pod_name> -n <namespace>
 ----
 
+.Testing
+
+The {ServerlessLogicOperatorName} automatically generates a route for workflow deployments using the `dev` profile. To test your workflow, you can use management console that is accessible via the created route. The management console allows you execute the workflow and inspect executed the instances in detail.
+
+. Access the management console by excuting following steps: 
+
+.. Retrieve the public URL to access the workflow service by running the following command:
++
+[source,terminal]
+----
+$ oc get route/<workflow_name> -n <namespace>
+----
++
+Note the value of `HOST/PORT` from the result of tis command, referenced as `<WORKFLOW_URL>` in next step.
```

Comment:
Typo x2:
```suggestion
Note the value of `HOST/PORT` from the result of this command, referenced as `<WORKFLOW_URL>` in the next step.
```
Also, if `<WORKFLOW_URL>` only appears as part of this documentation / procedure, could it be lowercase like a normal user-replaceable? E.g.:
```suggestion
Note the value of `HOST/PORT` from the result of this command, referenced as `<workflow_url>` in the next step.
```
If it has meaning as all-cap outside of this procedure, however (like in output or the UI someplace), then ignore.